app/models/user.rb

```diff
@@ -68,6 +68,8 @@ class User < ApplicationRecord
     s.string :twitter_oauth_token_secret
     s.string :twitter_username
     s.any :keybase_signatures, array: true
+    s.string :homepage
```

Comment:
??

Reply:
oh that was the conflicting line. i assumed it was supposed to exist.

Reply:
oh yes yes. i didn't realize it was previously deleted. 🆒